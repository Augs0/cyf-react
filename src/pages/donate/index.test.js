import React from 'react';
import { shallow } from 'enzyme';

import Donate from './';

describe('Donate Component', () => {
  it('renders', () => {
    expect(shallow(<Donate />)).toMatchSnapshot();
  });
});
