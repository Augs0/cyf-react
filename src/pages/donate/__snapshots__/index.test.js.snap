// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donate Component renders 1`] = `
<div>
  <Styled(div)>
    <Styled(div)>
      <DonationPicker
        onTokenCapture={[Function]}
      />
    </Styled(div)>
    <Styled(div)>
      <div>
        <h3>
          We work on a 100% donation model
        </h3>
        <br />
        <p>
          That means that all support from individuals will go entirely to support our students.
        </p>
        <p>
          This will help cover for childcare, transportation, laptops and internet for our students most in need.
        </p>
      </div>
    </Styled(div)>
  </Styled(div)>
  <Styled(div)>
    <img
      alt="code-your-future-mentors"
      src="mentors-group.jpg"
    />
  </Styled(div)>
</div>
`;
